src/Item/Item.errors.ts

```diff
@@ -28,6 +28,12 @@ export class ThirdPartyItemAlreadyPublishedError extends Error {
   }
 }
 
+export class URNAlreadyInUseError extends Error {
+  constructor(public id: string, public urn: string, action: ItemAction) {
+    super(`The selected URN is already being used. It can be ${action}.`)
```

Comment:
What do you think about this text?
```suggestion
    super(`The provided URN already belong to another item. The item can't be ${action}.`)
```